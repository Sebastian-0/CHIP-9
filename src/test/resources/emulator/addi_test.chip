ldi A, 0

# Zero flag
addi 0
sout FLAGS

# No flags
addi 1
sout FLAGS

# Half-carry flag
addi 15
sout FLAGS

# Negative flag
addi 112
sout FLAGS

# Carry flag
addi 255
sout FLAGS

# Total result
sout A

hcf