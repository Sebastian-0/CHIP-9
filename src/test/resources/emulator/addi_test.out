128
0
32
64
16
127